docs/src/process.rst

```diff
@@ -111,10 +111,38 @@ Data types
     ::
 
         typedef enum {
+            /*
+            * The following four options are mutually-exclusive, and define
+            * the operation to perform for the corresponding file descriptor
+            * in the child process:
+            */
+
+            /*
+            * No file descriptor will be provided (or redirected to
+            * `/dev/null` if it is fd 0, 1 or 2).
+            */
             UV_IGNORE = 0x00,
+
+            /*
+            * TODO(leonerd): I imagine this creates a pipe of some kind,
+            * giving one end to the created process. I have no idea what
+            * happens to the other end.
```

Comment:
```suggestion
            * Open a new pipe into `data.stream`, per the flags below. The
            * `data.stream` field must point to a uv_pipe_t object that has
            * been initialized with `uv_pipe_init(loop, data.stream, ipc);`,
            * but not yet opened or connected.
```

Reply:
bump? Could one of the other contributors make sure I didn't write something dumb here, and we can land this.

Reply:
Can someone proofread-check me here? We also need to retarget the PR to v1.x before landing.

Reply:
LGTM